engine/src/main/java/com/sandboni/core/engine/sta/executor/AbstractParallelExecutor.java

```diff
@@ -14,6 +14,7 @@
 
 public abstract class AbstractParallelExecutor<T, R> implements ParallelExecutor<Collection<T>, Collection<R>> {
 
+    private static final String EXECUTOR_NAME = "AbstractParallelExecutor";
```

Comment:
Believe there was a good reason for that, but why you can't use the getClass() name?